android/settings.gradle

```diff
@@ -3,5 +3,7 @@ include ':react-native-vector-icons'
 project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
 include ':react-native-vector-icons'
 project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-vector-icons'
```

Comment:
Same

Reply:
Thanks for noticing it, https://github.com/safely-app/Front-End-Mobile/pull/14/commits/70a0bf17e340d26abb15622750b38426a284ba68